### ISSUE TITLE
Using Expo

### ISSUE THREAD

**Issue by GlenneJSG** [mcnamee/react-native-starter-kit#120]
### Help us reproduce - tell us about your environment
1. Mac OS latest version
1.Cloned repository 28/10/17

### Steps to reproduce
trying to use expo to be able to see the app on my phone (iPhone 6) 
as detailed 
https://facebook.github.io/react-native/docs/getting-started.html

### Expected result

1. when npm start or yarn start is run in terminal it should produce a QR code for me to scan to see the app on my phone

### Actual result

This is what is happening instead
<img width="1030" alt="screen shot 2017-10-28 at 11 44 55" src="https://user-images.githubusercontent.com/17801959/32135972-756ba306-bbd5-11e7-96f6-aa1ce8467230.png">

**Comment by joeLloyd** [comment]
to use expo in this project do the following

change the start script to `"start": "react-native-scripts start"`

install the following packages 
```
expo,
react-native-scripts
```

Not sure if 100% necessary but I added a main in my package JSON
`"main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",`

also  make sure you have a `.expo` folder with `packager-info.json` & `settings.json`. this should be generated for you.

finally, make sure you have a app.json file, mine looks like this
```json
{
	"expo": {
		"sdkVersion": "21.0.0"
	}
}
```

After doing these things I was able to get the QR code up and running in my console.
